packages/cli/bin/cli.js

```diff
@@ -82,6 +82,18 @@ const argv = yargs
     describe: 'set log level to debug',
     type: 'boolean',
   })
+  .option('noHyperlinks', {
```

Comment:
`no-hyperlinks` and `no-colors` both errored as flags, so I changed them to camel case, and that fixed the issue.